includes/controls/hello-elementor-customizer-upsell.php

```diff
@@ -0,0 +1,107 @@
+<?php /**
```

Comment:
ABSPATH & Namespace?

Reply:
1. Remove the doc at the top of the file
2. Add the missing `ABSPATH` check
3. Add missing namespace `HelloElementor\Includes\Controls`

Reply:
Updated